mods/default/chests.lua

```diff
@@ -261,29 +262,31 @@ function default.chest.register_chest(name, d)
 	def_closed.tiles[5] = def.tiles[3] -- drawtype to make them match the mesh
 	def_closed.tiles[3] = def.tiles[3].."^[transformFX"
 
-	minetest.register_node("default:" .. name, def_closed)
-	minetest.register_node("default:" .. name .. "_open", def_opened)
+	minetest.register_node(prefixed_name, def_closed)
+	minetest.register_node(prefixed_name .. "_open", def_opened)
 
 	-- convert old chests to this new variant
-	minetest.register_lbm({
-		label = "update chests to opening chests",
-		name = "default:upgrade_" .. name .. "_v2",
-		nodenames = {"default:" .. name},
-		action = function(pos, node)
-			local meta = minetest.get_meta(pos)
-			meta:set_string("formspec", nil)
-			local inv = meta:get_inventory()
-			local list = inv:get_list("default:chest")
-			if list then
-				inv:set_size("main", 8*4)
-				inv:set_list("main", list)
-				inv:set_list("default:chest", nil)
+	if name:sub(1,8) == "default:" then
```

Comment:
Doesn't this only apply to `default:chest_locked` and `default:chest`? A check like this isn't safe in case MTG gets another type of chest.

Reply:
I do not know the background for this conversion. Do we need the LBM for all chests (1)? default chests (2)? Only the both mentioned chests (3)? Or is the conversion obsolete and could be removed (4)?

Current state is (2), you propose to change it to (3)? 
`if name == "default:chest" or name == "default:chest_locked" then`?

Reply:
The LBM here is intended to migrate old chests to the new ones which have an open lid. Since old worlds always require legacy code, this must be kept. I think that checking for (3) will suffice, for safety ask @sofar.